egs2/aphasiabank/asr1/README.md

```diff
@@ -9,6 +9,24 @@
 - Git hash: `39c1ec0509904f16ac36d25efc971e2a94ff781f`
     - Commit date: `Wed Dec 21 12:50:18 2022 -0500`
 
+## asr_train_asr_ebranchformer_small_wavlm_large1
+
+- [train_asr_ebranchformer_small_wavlm_large.yaml](conf/tuning/train_asr_ebranchformer_small_wavlm_large.yaml)
```

Comment:
Can you add a short note here to explain the difference? E.g., this config uses xxx as the input layer which does not perform downsampling.

Reply:
Is this config correct? It looks same as the previous one.

Reply:
Thanks! Fixed in 4734d75